docker-compose.prod.yml

```diff
@@ -0,0 +1,21 @@
+services:
+  nginx:
+    build:
+      context: nginx
+      dockerfile: Dockerfile
```

Comment:
i don't understand the Dockerfile. Couldn't we simply use a pre-built nginx container?